utils/export.go

```diff
@@ -2,6 +2,7 @@ package utils
 
```

Comment:
some functions are unused here (like `ExportOutputDir`, `ExportOutputFileContent`, `ExportOutputDirAsZip`, ...)
similar functionalities are shared here: https://github.com/bitrise-io/go-steputils/blob/master/output/output.go
I would try to remove this file and use the `go-steputils/output` package.

Reply:
That is a good tip. 👍🏻

Actually it was more interesting than I expected it to be. 😄  
The local `utils/export.go` was not used at all because this import `"github.com/bitrise-steplib/steps-xcode-archive/utils"` brought in exactly the same `export.go` file as the repo contained. It was using the function from there. 

Is it common to import features from another step in a step? Or this was just an error and common functionality should be moved in one of the utils packages?